CryptKeeperZK/crypt-keeper-extension
# feat: update identity creation

## Explanation

This PR improves identity creation process and adds deterministic values to metadata.

Details are below:
- [x] Add dropdown button component and update signature buttons
- [x] Add nonce for deterministic identities on identity screen
- [x] Minor test fixes and refactoring

## Related Issues

Related to #889 

## Screenshots

<details>
<summary>Expand</summary>

![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/7e3080db-2b44-4f40-b59a-9cc1450677c5)
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/4e18f51a-c497-4dbb-bc15-5114e7213ae5)

</details>


## Manual Testing Steps

Try to create identity with different signature options and check the identity page

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
